.nyc_output/out.json

```diff
@@ -94,6 +319,6 @@
     "f": {},
     "b": {},
     "_coverageSchema": "1a1c01bbd47fc00a2c39e90264f33305004495a9",
-    "hash": "7c8e731bd4cc34f700a6cad6ff919c39f9cdbffb"
+    "hash": "deaffac37b29f0c57fa7e42cf2ce61997b0700d0"
```

Comment:
This entire file needs to be removed from version control

Reply:
> This entire file needs to be removed from version control

removed now